libkineto/src/Config.cpp

```diff
@@ -42,6 +42,7 @@ constexpr int kDefaultSamplesPerReport(1);
 constexpr int kDefaultMaxEventProfilersPerGpu(1);
 constexpr int kDefaultEventProfilerHearbeatMonitorPeriod(0);
 constexpr seconds kMaxRequestAge(10);
+constexpr seconds kDefaultOnDemandConfigUpdateIntervalSecs(5);
```

Comment:
Why is this a different default value than the constexpr in configLoader.cpp:39 below? (ie. 5 vs 1)

Reply:
The one below is not even used, let me clean it up